packages/cubejs-databricks-jdbc-driver/src/DatabricksDriver.ts

```diff
@@ -733,6 +742,15 @@ export class DatabricksDriver extends JDBCDriver {
         url.host,
         objectSearchPrefix,
       );
+    } else if (this.config.bucketType === 'gcs') {
+      return this.extractFilesFromGCS(
+        { credentials: this.config.gcsCredentials },
+        url.host,
+        objectSearchPrefix+".csv",
```

Comment:
Is it important to add `.csv` here? Are there any other unrelated files that might be captured by `objectSearchPrefix` that should be excluded?

Reply:
Having this:
```js
  protected async extractFilesFromGCS(
    gcsConfig: GoogleStorageClientConfig,
    bucketName: string,
    tableName: string
  ): Promise<string[]> {
.......
const [files] = await bucket.getFiles({ prefix: `${tableName}/` });
```
Does Databricks create a folder for a table with a `csv` suffix? Like `my_exported_table.cvs/` with a bunch of real csv files?